skbio/stats/tests/test_composition.py

```diff
@@ -546,12 +546,197 @@ def test_ancom_basic_counts(self):
-        exp = pd.DataFrame({'W': np.array([5, 5, 2, 2, 2, 2, 2]),
-                            'reject': np.array([True, True, False, False,
-                                                False, False, False],
-                                               dtype=bool)})
+        exp = pd.DataFrame(
+            {'W': np.array([5, 5, 2, 2, 2, 2, 2]),
+             'Reject null hypothesis': np.array([True, True, False, False,
+                                                 False, False, False],
+                                                dtype=bool)})
+        assert_data_frame_almost_equal(result[0], exp)
+
+    def test_ancom_basic_counts_shuffled_order(self):
+        test_table = pd.DataFrame(self.table1)
+        original_table = copy.deepcopy(test_table)
+        test_cats = pd.Series(self.cats1)
+        # reverse the order of the grouping to confirm that
+        # order doesn't matter.
+        test_cats = test_cats.reindex(reversed(test_cats.index))
+        original_cats = copy.deepcopy(test_cats)
+        result = ancom(test_table,
+                       test_cats,
+                       multiple_comparisons_correction=None)
+        # Test to make sure that the input table hasn't be altered
+        assert_data_frame_almost_equal(original_table, test_table)
+        # Test to make sure that the input table hasn't be altered
+        pdt.assert_series_equal(original_cats, test_cats)
+        exp = pd.DataFrame(
+            {'W': np.array([5, 5, 2, 2, 2, 2, 2]),
+             'Reject null hypothesis': np.array([True, True, False, False,
+                                                 False, False, False],
+                                                dtype=bool)})
+        assert_data_frame_almost_equal(result[0], exp)
+
+    def test_ancom_percentiles(self):
+        table = pd.DataFrame([[12, 11],
+                              [9, 11],
+                              [1, 11],
+                              [22, 100],
+                              [20, 53],
+                              [23, 1]],
+                             index=['s1', 's2', 's3', 's4', 's5', 's6'],
+                             columns=['b1', 'b2'])
+        grouping = pd.Series(['a', 'a', 'a', 'b', 'b', 'b'],
+                             index=['s1', 's2', 's3', 's4', 's5', 's6'])
+
+        percentiles = [0.0, 25.0, 50.0, 75.0, 100.0]
+        groups = ['a', 'b']
+        tuples = [(p, g) for g in groups for p in percentiles]
+        exp_mi = pd.MultiIndex.from_tuples(tuples,
+                                           names=['Percentile', 'Group'])
+        exp_data = np.array(
+            [[1.0, 11.0], [5.0, 11.0], [9.0, 11.0], [10.5, 11.0], [12.0, 11.0],
+             [20.0, 1.0], [21.0, 27.0], [22.0, 53.0], [22.5, 76.5],
+             [23.0, 100.0]])
+        exp = pd.DataFrame(exp_data.T, columns=exp_mi, index=['b1', 'b2'])
+
+        result = ancom(table, grouping, percentiles=percentiles)[1]
```

Comment:
Can you have this test use the default percentiles (i.e. don't provide percentiles)? There currently isn't a test for that.